README.md

```diff
@@ -25,12 +25,13 @@ The version of this resource corresponds to the version of kubectl. We recommend
       - cluster:
         ...
     ```
+- `context`: *Optional.* The context to use when specifying a `kubeconfig` or `kubeconfig_file`
 
 ### cluster configs
 
 - `server`: *Optional.* The address and port of the API server. Requires `token`.
 - `token`: *Optional.* Bearer token for authentication to the API server. Requires `server`.
-- `namespace`: *Optional.* The namespace scope. Defaults to `default`.
+- `namespace`: *Optional.* The namespace scope. Defaults to `default`. If set along with `kubeconfig`, `namespace` will override the namespace in the current-context
```

Comment:
missing `.`

Reply:
I fixed it at f27931450ae3476b0f6652684f0ce86b45cba1ec.